marketplace-client/src/marketplace/src/app/_models/staff-member.ts

```diff
@@ -0,0 +1,12 @@
+import {Role} from './role';
+
+export interface StaffMember {
+  name?: string;
+  surname?: string;
+  password?: string;
+  email?: string;
+  dateOfBirth?: string;
+  phone?: string;
+  role?: Role;
+  status?: string;
+}
```

Comment:
why all the fields are optional? is it possible to create a user without, for example, name&